ngraph_bridge/ngraph_utils.cc

```diff
@@ -312,12 +312,15 @@ Status CheckAxisDimInRange(std::vector<int64> axes, size_t rank) {
 void NgraphSerialize(const std::string& file_name,
                      const std::shared_ptr<ngraph::Function>& ng_function) {
   NGRAPH_VLOG(0) << "Serializing graph to: " << file_name << std::endl;
-  std::string js = ngraph::serialize(ng_function, 4);
+  StringToFile(file_name, ngraph::serialize(ng_function, 4));
+}
+
+void StringToFile(const std::string& file_name, const std::string& contents) {
```

Comment:
Have we check if the dumped .json is able to load on nGraph?

Reply:
Checked the following:
```
NGRAPH_ENABLE_SERIALIZE=1 python mnist_deep_simplified.py
../../../nbench -f tf_function_ngraph_cluster_23.json -b CPU
```
which works

Reply:
Add a unit test to do the same if one doesn't exist.